docs/hardware_setup.md

```diff
@@ -23,7 +23,7 @@ The infrared cameras offer the option to run the KimeraVIO stereo version on mon
 
 7. Create configuration files for KimeraVIO ROS wrapper using [Kalibr2KimeraVIO-pinhole-radtan](https://github.com/MIT-SPARK/KimeraVIO/blob/feature/parallelization/jpl/kalibr/kalibr2sparkvio_stereo_pinhole-radtan.py)
 
-8. Create/adapt your own specific launch file, similar to [example RealSense IR](https://github.mit.edu/SPARK/spark_vio_ros/blob/jpl/launch/spark_vio_ros_realsense_IR.launch)
+8. Create/adapt your own specific launch file, similar to [example RealSense IR](https://github.com/MIT-SPARK/Kimera-VIO-ROS/kimera_ros_realsense_IR.launch)
```

Comment:
This returns "Not Found" for me. Probably should point to https://github.com/MIT-SPARK/Kimera-VIO-ROS/blob/master/launch/kimera_ros_realsense_IR.launch